onlineShop.core/Data/Entities/Cart.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace onlineShop.core.Entities
+{
+    public class Cart
+    {
+        [Key]
+        public string SessionId { get; set; }
+
+        public string _Products {
```

Comment:
Add some comments describing what is happening here.

Reply:
Done